setup.py

```diff
@@ -63,7 +63,7 @@
     entry_points="""
     [z3c.autoinclude.plugin]
     target = plone
-    [console_scripts]
+    [zopectl.command]
```

Comment:
I referred to this https://blog.niteoweb.com/write-a-plone-cli-maintenance-script/